src/components/Observations/map/ObservationMapView.tsx

```diff
@@ -163,9 +162,11 @@ export default function ObservationMapView({ observationsResults, search }: Obse
             }}
             highlightEntities={search === '' ? [] : searchZoneEntities}
             focusEntities={
-              search === '' || searchZoneEntities.length === 0
+              search === '' && searchZoneEntities.length === 0
```

Comment:
does it matter if the sites layer is off?

Reply:
Good question, it shouldn't cause a problem, but I'll test it out real quick to make sure

Reply:
@charlie-tf one small observation (pun intended).. when we search for zones it zooms into search zone (very cool 👍 ).. when we clear the search it does not zoom back out to the global bounding box.. possible to look into that?

Reply:
I had mostly provided the zooming in/out suggestions to Rachel.

Reply:
I can switch it back to zooming to encompassing the whole planting site if you think that's a better experience. Let's go with that and get this merged. I can change it back if I get a different answer

Reply:
Google maps does it the way you have it - you can probably just clarify/make this the new requirement!